posts/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+from users.models import User
+
+
+class Post(models.Model):
+    content = models.CharField(max_length=256)
+    author = models.ForeignKey(User)
```

Comment:
Yes, we need them, but it's the responsibility of the dev who implements those features to create the relationships.